src/animals/Antelope.js

```diff
@@ -0,0 +1,57 @@
+import { Animal } from '../organisms/Animal';
+import { getRandom } from '../utilities/getRandom';
+
+export class Antelope extends Animal {
+  constructor(positionY, positionX, age = 0) {
+    super(4, 4, positionY, positionX, age);
+  }
+
+  async action(board) {
```

Comment:
You could avoid overriding the `action` method by creating the `getAvailableDirections` and `shouldFight` methods and overriding them.